### DISCUSSION TITLE
Can't apply basic styles like margins or borders into the project

### DISCUSSION THREAD

**Discussion by kkdima** [tailwindlabs/tailwindcss#9467] [Help]
Hello everyone, I encountered the very annoying problem from the very beginning of the project.
In the nutshell I basically can't apply most of the styles provided by tailwindcss.
The boilerplate for the project was taken from the next-tailwindcss examples on GitHub.
Then added just one `ContactForm.js` page as component:

```
 import React, { useState } from 'react';

   const ContactForm = () => {
	const [status, setStatus] = useState('Submit');

	const handleSubmit = async (e) => {
		e.preventDefault();

		setStatus('Sending...');

		const { name, email, message } = e.target.elements;

		let details = {
			name: name.value,
			email: email.value,
			message: message.value,
		};

		let response = await fetch('http://localhost:5000/contact', {
			method: 'POST',
			headers: {
				'Content-Type': 'application/json;charset=utf-8',
			},
			body: JSON.stringify(details),
		});

		setStatus('Submit');

		let result = await response.json();

		alert(result.status);
	};

	return (
		<div className="mt-6">
			<form onSubmit={handleSubmit} className="text-white">
				<div className="flex flex-col mb-4">
					<label htmlFor="name">Name:</label>
					<input type="text" id="name" required />
				</div>

				<div className="flex flex-col">
					<label htmlFor="email">Email:</label>
					<input type="email" id="email" required />
				</div>

				<div className="flex flex-col">
					<label htmlFor="message">Message:</label>
					<textarea id="message" required />
				</div>

				<button type="submit" className="bg-blue">
					{status}
				</button>
			</form>
		</div>
	);
};

export default ContactForm;

```
The very first and second div element inside the component has some classnames as styles, but the thing is that only flex and flex-col apply. Stuff like margins doesn't render on the page at all.
Did anyone encounter stuff like this?

package.json file:
```
{
	"private": true,
	"scripts": {
		"dev": "next dev",
		"build": "next build",
		"start": "next start"
	},
	"dependencies": {
		"next": "latest",
		"react": "18.1.0",
		"react-dom": "18.1.0"
	},
	"devDependencies": {
		"@types/node": "17.0.35",
		"@types/react": "18.0.9",
		"@types/react-dom": "18.0.5",
		"autoprefixer": "^10.4.7",
		"postcss": "^8.4.14",
		"tailwindcss": "^3.1.8",
		"typescript": "4.7.2"
	}
}

```

**Comment by wongjn** [comment]
I'm not sure what these "next-tailwindcss examples on GitHub" are, but have you tried following the [Next.js integration guide](https://tailwindcss.com/docs/guides/nextjs)? Further, is the ContactForm.js file covered by your `content` file globs in your Tailwind configuration? Perhaps a full repository that exhibits your issue would also help in troubleshooting if you are able to share it.

**Reply by kkdima** [reply]
Thank you for reminding about their purge feature. I had to edit the path the the my components folder because I put it inside `src` folder. Now it works, thank you.

I was referring to the set of [nextjs's boilerplates](https://github.com/vercel/next.js/tree/deprecated-main/examples) to start project with everything installed depending what technology you want to use. In my case I started with nextjs + tailwindcss template.
